simpler_setup/tools/strace_timing.py

```
@@ -396,6 +409,167 @@ def to_chrome_trace(invocations, buckets=None):
     return {"traceEvents": events, "displayTimeUnit": "ms"}
 
 
+def _parsed_attrs(span):
+    attrs = {}
+    for field in span.attrs.split():
```

Ruff (0.16.1)

[error] Import `field` from line 51 shadowed by loop variable

(F402)